extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+# 3DTILES_implicit_tiling Extension
```

Comment:
There should be some mention of how `3DTILES_implicit_tiling` and a tileset's `root` property interact.

Reply:
Added rule in Root Tiles section.